css/style.css

```diff
@@ -845,17 +854,11 @@ button:hover {
 
 @media (max-width: 480px) {
 	/* hide board actions on mobile */
-	#board-actions {
+	#board-header-controls {
 		display: none !important;
```

Comment:
found a bug @artemanufrij  - clicking on the "Add new stack" immediately closes the popover...